### PR TITLE
Nerfs confusion

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request
Random direction --> maximum 50 chance for random direction when over 40
Compounded 1.5 * confusion chance for turning 90 degrees to just confusion
Compounded 3 * confusion chance for turning 45 degrees to just confusion * 2

## Why It's Good For The Game

Confusion's ridiculously strong. It's an interesting effect but it shouldn't nullify movement.

This should probably be made into some sort of status effect in the future but for the moment I think this is a fair change that would allow more usage of it without making it completely debilitating.

Probably needs a testmerge.

## Changelog
:cl:
balance: Confusion effects have been significantly nerfed.
/:cl:

### CODE REVIEW

**Comment by Ludox235** [comment]
Nobody asked for this :/

**Comment by Ghommie** [comment]
Confusion being ridicolously strong is actually an interesting factor, it doesn't nullify movement, but still severely restricts people from gracefully waltzing around halls, though it seems to last quite a bit too much for things like flashes or reagents, they are not designed that well.

TL;DR: Sumotori Dreams was a funny game.

**Comment by TooFewSecrets** [comment]
Specifically, Changeling REEEEEEing totally shutting movement down in an AOE for a full minute through bowman headsets is stupid. Also applies to most other sources; confusion is a very drive-by status effect for how much it cripples you, and not exactly hard to get access to.

**Review by Ghommie — APPROVED** [review]
